budcamp/Homework2/Homework2/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+
+namespace Homework2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Task1();
+
+            Task2();
+
+            Task3();
+
+            Task4();
+
+            Task5();
+
+            Task6();
+        }
+
+        private static void Task1()
+        {
+            Marker markerGreen = new Marker();
+            markerGreen.Color = "green";
+
+            Marker markerOrange = new Marker();
+            markerOrange.Color = "orange";
+
+            markerGreen.PrintMarker();
+            markerOrange.PrintMarker();
+
+            Marker? markerBlue = Marker.InitMarker(null);
+            markerBlue ??= new Marker("default");
+            markerBlue?.PrintMarker();
+        }
+
+        private static void Task2()
+        {
+            Book bookBasic = new Book();
+            bookBasic.Title = "BASIC";
+            bookBasic.Author = "S.TROELSTRA";
+
+            Book bookC = new Book();
+            bookC.Title = "C+";
+            bookC.Author = "G.RTRTG";
+
+            bookBasic.PrintBookInfo();
+            bookC.PrintBookInfo();
+        }
+
+        private static void Task3()
+        {
+            Console.WriteLine("Input the number of elements to store in the array :");
+            int arraySize = int.Parse(Console.ReadLine());
```

Comment:
if you don't use `int.TryParse` or `Convert.ToInt32` you need to handle possible exceptions.

Reply:
i knew... but i thought it is not a priority for the task. Tried to do pure logic that need, without handling possible issues. In any case, fixed

Reply:
can't see how it fixed

Reply:
i'll push it when all the comments wil be done